### DISCUSSION TITLE
Pregunta Task 1

### DISCUSSION THREAD

**Discussion by Ricardo-OB** [rramosp/ai4eng.v1#6] [04 - GESTIÓN DE DATOS]
Hola profe Raul,
Ya tengo el dataframe con los datos correspondientes (LAB 04. 02, Task 1), a qué hace referencia con:

4. Extract the column of the target variable and sum all values.
5. Sum all the values of the rest of the columns.

No me queda muy claro los pasos a seguir, gracias, quedo atento.

**Comment by rramosp** [comment]
Hola Ricardo, 

fíjate que tienes que construir un dataset con las siguientes columnas: ‘meter_reading’, ‘site_id’, ‘air_temperature’, ‘cloud_coverage’, ‘dew_temperature’, ‘precip_depth_1_hr’, ‘sea_level_pressure’, ‘wind_direction’, ‘wind_speed’, ‘square_feet’, ‘year_built’  ... integrando datos de los tres ficheros csv que se componen el challenge.

pero sólo para el día y edificio que se te indica y usando solo las filas con meter=0

en el caso del edificio número 900 y de la fecha 2016-02-01 te quedaría un dataset con 24 filas y las columnas anteriores.

el **target variable** es lo que se pretende predecir, en este caso es la columna de la lectura del medidor (**meter_reading**). tienes que sumar todos los valores de esa columna (sumarías 24 valores que darían un total 4941 en el caso del edificio 900 y la 2016-02-01)

luego, tienes que sumar los valores del resto de las columnas del dataset que armaste (todas expecto **meter_reading**.

espero que con esto esté más claro. te recomiendo que trates de hacer el ejemplo del edificion 900 y la fecha indicada para convencerte de que entendiste bien el ejercicio.

**Reply by Ricardo-OB** [reply]
Todo claro, ya lo confirme con los datos de ejemplo que me dice.
Muchas gracias.

**Reply by rramosp** [reply]
perfecto!!!

**Reply by pymilo** [reply]
Hola Raul,

También estuve confundido un buen rato con este problema. Creo que en las instrucciones que aparecen en el Notebook les falta mencionar que para esta parte del ejercicio la **target variable** es la lectura del medidor **meter_reading**. Sería bueno editar esa parte si fuese posible para los estudiantes que se encuentren con este problema en el futuro. Muchas gracias por publicar esto aquí. En temas como este es que se evidencia la utilidad de los foros 🤓.
